packages/toolpad-studio/src/toolpad/AppEditor/PageEditor/PageOptionsPanel.tsx

```diff
@@ -122,7 +122,38 @@ export default function PageOptionsPanel() {
         </Tooltip>
       </div>
       <div>
-        <Typography variant="body2">Authorization:</Typography>
+        <Typography variant="overline">
+          Authorization{' '}
+          {!isPaidPlan ? (
+            <Tooltip
+              title={
+                <Typography variant="inherit">
+                  Only available in paid plans. Click to learn more.
+                </Typography>
+              }
+            >
+              <Chip
+                variant="outlined"
+                color="primary"
+                component={'a'}
```

Comment:
I wish prettier would fix these

```suggestion
                component="a"
```